gen3sdk/R/Gen3AuthHelper.R

```diff
@@ -0,0 +1,63 @@
+library(httr)
+library(jsonlite)
+
+Gen3AuthHelper <- setRefClass("Gen3AuthHelper",
+
+# Gen3 auth helper class for use with requests auth.
```

Comment:
there's a [particular way](http://r-pkgs.had.co.nz/man.html#man-classes) you can add documentation for RC classes, so like here you can add `@field endpoint` etc